### PR TITLE
Minor indentation in README.md

### PR DESCRIPTION
**Pull request by kimegede**
Just a minor indentation in the documentation.

### CODE REVIEW

**Review by alvarocastro — APPROVED** [review]

**Review by sathishsoundharajan — APPROVED** [review]

**Comment by Glo753** [comment]
npm install @codertocat/hello-world-npm@1.0.1

**Review by Glo753 — APPROVED** [review]

**Review by Glo753 — APPROVED** [review]

**Review by alphaolomi — APPROVED** [review]

**Comment by Katay88** [comment]
Good first issues

 Document edited status

 Keep filename label (and progress spinner) visible

**Comment by Katay88** [comment]
patchURL

**Review by Katay88 — APPROVED** [review]

**Comment by Ledear-saaano** [comment]
تثبيت npm @ codertocat / hello-world-npm @ 1.0.1

**Review by Tylorriviere — COMMENTED** [review]
Yes this is the best stuff ever

**Comment by Tylorriviere** [comment]
This message is intended for the money to pay for the money to the guests in the morning

**Review by Tylorriviere — COMMENTED** [review]
The New school for the money

**Comment by Tylorriviere** [comment]
README.md

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Or add this package to your `package.json` file:
 ```
 "dependencies": {
     "@codertocat/hello-world-npm": "1.0.0"
```
Comment (kimegede) [inline README.md]:
```suggestion
    "@codertocat/hello-world-npm": "1.0.2"
```

**Comment by kimegede** [comment]
Updated

**Review by Tylorriviere — APPROVED** [review]
Thank you for the help my fellow friends

**Review by Andipandi111 — APPROVED** [review]

**Review by Andipandi111 — APPROVED** [review]

**Review by Andipandi111 — APPROVED** [review]

**Comment by kimegede** [comment]
Should we close this one?

**Pull request closed without merging**